integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/AnsibleResource.java

```diff
@@ -0,0 +1,76 @@
+package com.redhat.service.smartevents.integration.tests.resources;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static io.restassured.RestAssured.given;
+
+public class AnsibleResource {
+
+    private static final String ANSIBLE_ENDPOINT = System.getProperty("ansible.endpoint") + "/api/v2";
+    private static final String ANSIBLE_USERNAME = System.getProperty("ansible.username");
+    private static final String ANSIBLE_PASSWORD = System.getProperty("ansible.password");
+
+    public static boolean jobTemplateExists(String name) {
+        final int templateCounts = given()
+                .baseUri(ANSIBLE_ENDPOINT)
+                .auth().preemptive().basic(ANSIBLE_USERNAME, ANSIBLE_PASSWORD)
+                .relaxedHTTPSValidation()
+                .basePath("job_templates")
+                .queryParam("name", name)
+                .when()
+                .get()
+                .then()
+                .statusCode(200)
+                .extract()
+                .jsonPath()
+                .getInt("count");
+
+        if (templateCounts > 0) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public static int jobTemplateID(String name) {
+        final int templateID = given()
+                .baseUri(ANSIBLE_ENDPOINT)
+                .auth().preemptive().basic(ANSIBLE_USERNAME, ANSIBLE_PASSWORD)
+                .relaxedHTTPSValidation()
+                .basePath("job_templates")
+                .queryParam("name", name)
+                .when()
+                .get()
+                .then()
+                .statusCode(200)
+                .extract()
+                .jsonPath()
+                .getInt("results.id[0]");
+
+        return templateID;
+    }
+
+    public static String jobs(int jobTemplateId, int days, int maxItems) {
+        final LocalDate now = LocalDate.now();
+        final LocalDate startTime = now.minusDays(days);
```

Comment:
what about timezones? https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/584f85ae461e614f99e25484032c44b4edca25bd/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/SlackResource.java#L22

Reply:
It is the reason why I check the jobs which run during the last two days. Therefore I do not expect any problems with time zones.